src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -55,13 +55,23 @@
 public final class FindBreakpointEvidenceSpark extends GATKSparkTool {
     private static final long serialVersionUID = 1L;
 
+    private static final String OUTPUT_ORDER_SHORT_NAME = "sort";
+    private static final String OUTPUT_ORDER_FULL_NAME = "outputSortOrder";
+
     @ArgumentCollection
     private final FindBreakpointEvidenceSparkArgumentCollection params =
             new FindBreakpointEvidenceSparkArgumentCollection();
 
     @Argument(doc = "sam file for aligned contigs", shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
             fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME)
-    private String outputSAM;
+    private String outputAssemblies;
```

Comment:
What about ```outputAssemblyAlignments```?, the reason I don't like outputSAM is because implies that the output format is ```SAM``` but it could be ```BAM``` or even ```CRAM``` in the future.